### PR TITLE
[MTDSA-4738] ITs no longer run UTs

### PR DESCRIPTION
**Pull request by james-work-account**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -114,6 +114,11 @@ api {
 httpHeadersWhitelist = [ "Accept", "Gov-Test-Scenario", "Content-Type", "Location", "X-Request-Timestamp", "X-Session-Id" ]
 internalServiceHostPatterns = [ "localhost" ]
 
+mongo-async-driver.akka {
+  log-dead-letters = off
+  log-dead-letters-during-shutdown = off
+}
+
```
Comment (BetaDraconis) [inline conf/application.conf]: This project doesn't actually implement Mongo (it was going to at one point but it never actually got implemented). Is this change required?

Comment (james-work-account) [reply conf/application.conf]: I've removed this and other references to mongo.
